### PR TITLE
Update to AWS Provider 3.x

### LINKED ISSUE #12: Terraform 0.12 Support
Currently the module is inoperable with Terraform 0.12 code, it would be great if it could support Terraform 0.12 :)

### LINKED ISSUE #24: 3.x AWS provider not supported
## Describe the Bug
The module requires version 2.x of the AWS provider, which is no longer under active development.

## Expected Behavior
If included in a project that uses version 3.x of the AWS provider, the module should work properly.

## Steps to Reproduce
Reference the module from a project with a newer version of the AWS provider, e.g., one that has

```
terraform {
  required_providers {
    aws = {
      source  = "aws"
      version = "~> 3.10"
    }
  }
}
```

Running `terraform init` will print an error because it can't satisfy the version constraints of the project and the module at the same time.

## Environment (please complete the following information):
Terraform 0.13.4, AWS provider 3.10.0.

## Additional Context
I tried modifying a local copy of the module to relax the version constraint (changed it from `~> 2.0` to `>= 2.0`) and it appeared to work fine with provider version 3.10.0, though I didn't test every possible permutation of inputs.

### PR DESCRIPTION
**Pull request by dmitrijn**
## what
* Update AWS provider version

## why
* Can't use this module with aws 3.x provider

## ref
* closes #24 
* closes #12

### CODE REVIEW

File: .github/workflows/chatops.yml [hunk 1]
```diff
@@ -9,13 +9,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: "Handle common commands"
-        uses: cloudposse/actions/github/slash-command-dispatch@0.15.0
+        uses: cloudposse/actions/github/slash-command-dispatch@0.16.0
```
Comment (joe-niland) [inline .github/workflows/chatops.yml:12]: Please use version `0.21.0`

Example: https://github.com/cloudposse/terraform-aws-ecs-container-definition/blob/master/.github/workflows/chatops.yml#L12

Comment (dmitrijn) [reply .github/workflows/chatops.yml]: fixed

**Review by joe-niland — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks very much @dmitrijn !

**Pull request merged**